documentation/modules/installing-mtv-operator.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 
 .Prerequisites
 
-* {ocp} {ocp-version} or later installed.
+* {ocp} 4.14 or 4.15 later installed.
```

Comment:
ah ok, I referred to this part in my previous comment about `ocp-version` - I think it's better to keep referencing `ocp-version` and set it to the earliest supported version, @RichardHoch what do you think?

Reply:
@ahadas Agreed .